{"uuid":"d8417db0-d4df-4b47-9d39-adc30d380a61","name":"Default test","children":["bc2ee158-804e-464b-844b-1118b706224e","62fdab4d-68aa-4f59-9683-b27f9977291b","323c0623-516a-444a-b745-5656132ce577","9e7f4066-0908-4941-8292-f5afa64f45f6","bd5c5fba-afd0-4a19-aba6-23c54a83119e","e0d1ec8d-d46c-46c0-82d3-cf5eb26eb968","4c35becc-9ba3-48de-b62e-063eb60e4d3a","980fe424-d257-436d-ae96-c7b80503318f"],"befores":[{"name":"setUp","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"chromedriver located at path/to/chromedriver.exe, but invalid\nFor documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location/\nBuild info: version: '4.12.1', revision: '8e34639b11'\nSystem info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.8.1'\nDriver info: driver.version: ChromeDriver","trace":"org.openqa.selenium.remote.NoSuchDriverException: chromedriver located at path/to/chromedriver.exe, but invalid\nFor documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location/\nBuild info: version: '4.12.1', revision: '8e34639b11'\nSystem info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.8.1'\nDriver info: driver.version: ChromeDriver\r\n\tat org.openqa.selenium.remote.service.DriverFinder.getPath(DriverFinder.java:45)\r\n\tat org.openqa.selenium.remote.service.DriverFinder.getPath(DriverFinder.java:13)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.generateExecutor(ChromeDriver.java:99)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:88)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:83)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:51)\r\n\tat runner.FeatureRunner.setUp(FeatureRunner.java:21)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)\r\n\tat org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:663)\r\n\tat org.testng.TestRunner.beforeRun(TestRunner.java:652)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1200)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1114)\r\n\tat org.testng.TestNG.run(TestNG.java:1082)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1699668629225,"stop":1699668629413}],"afters":[{"name":"tearDown","status":"broken","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1699668630533,"stop":1699668630533}],"start":1699668629148,"stop":1699668630539}